pkg/tmpl/context.go

```diff
@@ -7,7 +7,9 @@ import (
 type Context struct {
 	preRender bool
 	basePath  string
-	fs        *filesystem.FileSystem
+	// directory of the value of --file/-f option
+	rootDir string
```

Comment:
I thought basePath was `path/to` part of the `path/to/helmfile.yaml`.
Probably we don't need to introduce a new `rootDir` and just use `basePath` all over there?!

Reply:
@mumoshu At first, I thought about it the same way, but during the helmfile processing, there will be `chdir` operations involved, so recording a `rootDir` will be clearer.

Reply:
@yxxhero As far as I remember, `basePath` always point to the directory where the helmfile.yaml is located. It should be true even for nested-helmfiles as each nested-helmfile has its own State and therefore tmpl.Context.

So not only this is a bit incorrect, it looks like this is complicating things without no benefit!
You could just use `basePath` instead of RootDir so that this becomes better with less code.